src/Tribe/Editor/Blocks/Tickets.php

```diff
@@ -103,7 +103,7 @@ public function assets() {
 			$plugin,
 			'tribe-tickets-gutenberg-tickets',
 			'tickets-block.js',
-			[ 'jquery', 'jquery-ui-datepicker', 'wp-util-not-in-footer', 'wp-i18n' ],
+			[ 'jquery', 'wp-util-not-in-footer', 'wp-i18n' ],
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 580fb7b9af81a0e18bdf28d276d496c1</sub>